include/seqan3/core/platform.hpp

```diff
@@ -189,9 +189,27 @@
 //  Deprecation Messages
 // ============================================================================
 
+//!\brief _Pragma requires a string-literal and # makes it a string
+#ifndef SEQAN3_PRAGMA
+#   define SEQAN3_PRAGMA(non_string_literal) _Pragma(#non_string_literal)
```

Comment:
_Pragma is portable between at least gcc and clang?

Reply:
yes